### PR TITLE
README.md Updated

### PR DESCRIPTION
**Pull request by AndriyVel05**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,59 @@
...
-- мати можливість вибирати автора імплементація якого буде ранитись
-- мати можливість переглянути список задач
-- мати можливість заранити задачу
-- мати можливість ввести необхідні дані
\ No newline at end of file
+## Description
+This project is a console application written in Java that allows running specific tasks collected from the Codewars platform. The application provides an interface for selecting a task author and executing their method to perform a specific task.
+
+## Requirements
+- Java 8 or later
```
Comment (smelovd) [inline README.md:7]: remove section

File: README.md [hunk 2]
```diff
@@ -1,7 +1,59 @@
...
+        2. View the list of tasks.
+        3. Run an author’s method to execute a task.
+
+3. **Executing a Task:**
+    - The user selects an author, a task, enters the required parameters, after which the task is executed, and the result is displayed in the console.
+
+## Example Usage
+
+To run the program, follow these steps:
+
```
Comment (smelovd) [inline README.md:42]: remove section

File: README.md [hunk 3]
```diff
@@ -1,7 +1,59 @@
...
+1. Open your IDE and make sure you have Java 8 or later installed.
+2. Import the project and locate the `UIController` class.
+3. Run the `UIController` class.
+4. Follow the on-screen prompts to select an author and task, enter parameters, and execute the task.
+
+## Contributing
+
+If you would like to contribute to this project, please follow these steps:
+
```
Comment (smelovd) [inline README.md:51]: remove section

File: README.md [hunk 4]
```diff
@@ -1,7 +1,59 @@
...
+- **UIController:** Responsible for interacting with the user through the console, handling user input, and invoking corresponding actions.
+- **AuthorController:** Manages operations related to authors (retrieving the list of authors, selecting a specific author).
+- **TaskController:** Manages the execution of tasks selected by the user.
+- **Services (AuthorService, TaskService):** Provide the logic for working with authors and tasks.
+- **Repositories (AuthorRepository, TaskRepository):** Store data about authors and tasks.
+
+## Usage
+
+1. **Running the Application:**
+    - Launch the `UIController` in your IDE.
```
Comment (smelovd) [inline README.md:28]: remove line

File: README.md [hunk 5]
```diff
@@ -1,7 +1,59 @@
...
\ No newline at end of file
+## Description
+This project is a console application written in Java that allows running specific tasks collected from the Codewars platform. The application provides an interface for selecting a task author and executing their method to perform a specific task.
+
+## Requirements
+- Java 8 or later
+- IDE to run the application (e.g., IntelliJ IDEA, Eclipse)
+
+## Architecture
+
```
Comment (smelovd) [inline README.md:11]: add image

**Review by smelovd — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (AndriyVel05) [reply README.md:7]: Java 8 or later removed.

**Pull request closed without merging**